rules/wow.ts

```diff
@@ -0,0 +1,15 @@
+import * as YAML from 'yaml';
+
+import { defineRule } from '../api/api.ts';
+import { LOCATION } from '../core/rule.state.ts';
+
+export default defineRule({
+	id: 'wow',
+	progress: async ({ stаte }) => {
+		const totalProgresses = stаte?.totalProgresses || 0;
+		const state = { totalProgresses: totalProgresses + 1, ...stаte };
+
+		await Deno.writeTextFile(LOCATION, YAML.stringify(state));
+		console.log('wow');
+	},
+});
```

Comment:
If we're adding something to the core state yaml, we should implement that in the core rule.

Als we should add new properties to the core state validator:
https://github.com/ndelangen/nomic/blob/1346d0620fb6e423eeb8dbff878911db47d1fedd/core/rule.state.ts#L7-L13

Rules run in a non-deterministic order, so there's the possibility of a race-condition here, where `state/core.yml` gets written to multiple times.

There's a type issue:
<img width="588" alt="Screenshot 2024-03-14 at 21 00 22" src="https://github.com/ndelangen/nomic/assets/3070389/aeef4c00-6b88-4c84-9440-2fea0c840972">

Is this `a` character something intentional?
<img width="904" alt="Screenshot 2024-03-14 at 21 01 35" src="https://github.com/ndelangen/nomic/assets/3070389/fbce2a2c-c58c-40c6-b6d9-7035faafcf84">